app/src/main/java/com/kickstarter/viewmodels/BackingAddOnsFragmentViewModel.kt

```diff
@@ -252,12 +257,17 @@ class BackingAddOnsFragmentViewModel {
                                 .build()
                     }
                 }
-
                 return@combineLatest Pair(updatedPledgeData, pledgeReason)
             }
+
+            this.continueButtonPressed
+                    .compose<Pair<Boolean, Pair<PledgeData, PledgeReason>>>(combineLatestPair(pledgeDataAndReason))
+                    .filter { it.first }
```

Comment:
Fixed an issue where the user selects an add-on, presses continue, then goes back to the add-ons screen and changes their add-on selection. When the add-ons selection is changed, the view automatically transitions to the pledge screen. The transition should only happen when selecting the continue button

Reply:
that one is also fixed on my PR, I'll add you to it